### ISSUE TITLE
Retrieving data from columns of table which contain html tags in them. 

### ISSUE THREAD

**Issue by SaurabhKulkarni21** [olifolkerd/tabulator#1632]
Hi Oli, 

I really love using tabulator and it has made complicated things a lot simpler. Recently i was trying to extract values from the table using getData to check for data of a specific column which contains an html tag in it, which i have done using "tabulator-formatter="html". It seems that i get the html tag in the string format rather than just the data that it contains. I am not sure if its a bug or just the way tabulator should work. 

Also, I was just wondering how i can create data tree for a specific row if i have initiated tabulator using an html table. 

I would like to get your help regarding this and I really appreciate your efforts.

Thanks,
Saurabh

**Comment by olifolkerd** [comment]
Hey @SaurabhKulkarni21 

Thanks for getting in touch, sorry for the delayed reply, work has been keeping me busy this week.

Im afraid without an example of the html for your ```tabulator-formatter="html``` issue i would be unable to offer advice. if you could include an html snippet of your table that would be great.

As for using ***dataTree*** with an html table, that one would be tricky. The best approach i can think of you would be include the child row data as JSON in a cell and then use a ***mutator*** to convert that into an array for Tabulator to extract the children.

If that is something you need an example of let me know and i can put something together for you.

Cheers

Oli :)

**Comment by SaurabhKulkarni21** [comment]
Hi Oli,

Sorry for the delays in replying.

So the issue that i was stuck on has been solved. It was something like how i can add child elements to a table which has been constructed using html with tabulator. 

So this is how my approach was:
1)Get the data after the object for the tabulator has been created. 
2) Manipulate the data (Add the child elements to which ever rows i want and also delete the child elements to avoid repetition/duplication)
3) Set the data into the table again. 

If there is something better than this method i would really appreciate it. 

Also, right now i am stuck on how i can increase the pagination size whenever the child row has been expanded. I went through the source code of the example that you guys have put up for data tree but could not find the solution. So i would really appreciate if you would provide any insights on this.

Thanks,
Saurabh
